src/core/tests/type_prop/reverse_sequence.cpp

```diff
@@ -87,7 +87,8 @@ TEST(type_prop, reverse_sequence_invalid_data_rank) {
             FAIL() << "Invalid rank of data input not detected";
         } catch (const NodeValidationFailure& error) {
             EXPECT_HAS_SUBSTRING(error.what(), std::string("Data input rank should be equal or greater than 2."));
-        } catch (...) {
+        } catch (std::exception& ex) {
+            std::cout << ex.what() << std::endl;
```

Comment:
Could you please explain why we need this change?

Reply:
sorry, it's for debug, I will remove them